.ci/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+TAG=$(echo $GIT_COMMIT | cut -c1-7)
+docker login -u ${DOCKER_HUB_LOGIN} -p ${DOCKER_HUB_PASSWORD}
+
+# Run tests and all the other checks on repo
+docker build -f build/CI/Dockerfile .
+
+# Unified Plugin Broker
+docker build -t eclipse/che-unified-plugin-broker:latest -f build/unified/Dockerfile .
```

Comment:
please fix ctx and paths

Reply:
Should be better now. Tested locally without docker login and push commands and it works.